amr-wind/incflo.cpp

```diff
@@ -310,6 +311,9 @@ void incflo::Evolve()
 
 void incflo::do_advance()
 {
+    if (m_sim.has_overset()) {
+        m_ovst_ops.pre_advance_actions();
```

Comment:
we have a bunch of places with `pre_advance_work`. Should this be renamed to that? And should it actually be moved to `pre_advance_stage2` where we do all the pre_advance stuff?

Reply:
Renaming it sounds good. The difficulty with putting it in either pre_advance_stage1 or pre_advance_stage2 is that these are actions that need to take place after the exchange_solution step is performed by exawind-driver. Both the pre advance stages are called prior to exchange_solution. If we wanted to, having another pre_advance_stage would be another way to organize it, but that seems like too much, especially because I don't think there's a nalu-wind counterpart for that.

Reply:
ah that's a good reason